egui_web/src/webgl2.rs

```diff
@@ -192,7 +192,17 @@ impl WebGl2Painter {
             }
         }
     }
-
```

Comment:
next time please don't remove the blank lines between function. I like them for readability :)